administrator/language/de-DE/com_installer.ini

```
@@ -72,6 +72,7 @@ COM_INSTALLER_HEADING_PACKAGE_ID="Paket-ID"
 COM_INSTALLER_HEADING_PACKAGE_ID_ASC="Paket-ID aufsteigend"
 COM_INSTALLER_HEADING_PACKAGE_ID_DESC="Paket-ID absteigend"
 COM_INSTALLER_HEADING_PROBLEMS="Probleme"
+COM_INSTALLER_HEADING_SUPPORTED="Download-Schlüssel"
```

Hmm ist es nich eher ein Lizenzschlüssel?

```suggestion
COM_INSTALLER_HEADING_SUPPORTED="Lizenzschlüssel"
```

---

Wir verwenden bisher nur Download-Schlüssel überall.

---

Und ein Licence Key vs Download Key hätte im Core bestimmt auch was anderes zu bedeuten?